keen/explorer
# Fixes #85

## What does this PR do? How does it affect users?
Ensures results with a value of 0 properly show a metric.

## How should this be tested?

Try running a query that you know for sure will return a result of 0. Here's a good example:

1) Run a query for an event that happens infrequently and set the timeframe to `this_1_minutes`.
2) Observe that the chart area shows a metric with a value of 0 instead of a message of "no results".

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?
#85
